virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -6,21 +6,30 @@ toc::[]
 
 Before you install {VirtProductName}, ensure that your {product-title} cluster meets the following requirements:
 
-* Your cluster must be installed on
-xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[bare metal] infrastructure with Red Hat Enterprise Linux CoreOS (RHCOS) workers.
+* Your cluster must be installed using any kind of
```

Comment:
Suggestion: "Your cluster must be installed on bare metal infrastructure with Red Hat Enterprise Linux CoreOS (RHCOS) workers. You can use any installation method including user-provisioned, installer-provisioned, or assisted installer to deploy your cluster."